packages/pages/src/components/analytics/provider.tsx

```diff
@@ -38,7 +43,8 @@ export function AnalyticsProvider(
 
   let enableDebuggingDefault = debuggingParamDetected();
   if (typeof process !== "undefined") {
-    enableDebuggingDefault = enableDebuggingDefault || process.env?.NODE_ENV === "development";
+    enableDebuggingDefault =
```

Comment:
Wondering why this got reformatted. There's a commit hook to run prettier. So either it didn't work for you or it didn't work for Tom, which is weird.

Reply:
I actually ran prettier manually before committing, and it made this change